changelog/unreleased/kong/fix-acme-renewal-bug.yml

```diff
@@ -0,0 +1,3 @@
+message: "**ACME**: fix an issue where using client.renew_certificate directly as the callback function in ngx_timer_at causes the parameter value to not be the plugin's config."
```

Comment:
The message is too technical, could we change it more understandable?

Reply:
Agreed, we can write technical into the commit message, but the changelog is not a good place for that.

Reply:
Agreed, we can write technical words into the commit message, but the changelog is not a good place for that.

Reply:
How about this one?

> "**ACME**: fix an issue where the certificate was not successfully renewed during ACME renewal."

It's already updated in changelog.